internal/transform/span_test.go

```diff
@@ -359,9 +356,11 @@ func TestTransform(t *testing.T) {
 		if lumigoSpan.LambdaType == "http" {
 			lumigoSpan.ID = mockLambdaContext.AwsRequestID
 		}
-		if !reflect.DeepEqual(lumigoSpan, tc.expect) {
-			t.Errorf("%s: %#v != %#v", tc.testname, lumigoSpan, tc.expect)
+		// assert.Equal(t, tc.expect, lumigoSpan)
+		if diff := cmp.Diff(tc.expect, lumigoSpan); diff != "" {
+			t.Errorf("%s mismatch (-want +got):\n%s", tc.testname, diff)
```

Comment:
What's the difference between the previous section?

Reply:
this is more descriptive and works as expected (brought to the world by google)